utils/exceptions.cc

```diff
@@ -73,3 +74,25 @@ bool is_timeout_exception(std::exception_ptr e) {
     }
     return false;
 }
+
+#if defined(__GLIBCXX__)
+
+#include <typeinfo>
+#include "utils/abi/eh_ia64.hh"
```

Comment:
Looks like we should also add another #ifdef for checking target architecture and fall back to previous code if it isn't compatible. Scylla also compiles on aarch64, which might follow a different ABI? In which case we should neither compile these snippets nor enable throwless catch in the first place.

Reply:
We should probably opt-in to the optimization by arch here.

But:
 - please check aaaaaaaaaaaaaarch64 now and opt-in if it works
 - emit a #warn for archs that didn't opt-in, since they will suffer significant degradation (we have -Wno-error=cpp)